### DISCUSSION TITLE
Tips on how to improve with Helix

### DISCUSSION THREAD

**Discussion by ThomasAlban** [helix-editor/helix#7276] [General]
Hi,
I'm very new to the concept of modal editors, and I thought Helix looked like a really good way to start. I've been using it for a few hours having learnt some of the basic keybinds, but am really struggling to focus on anything apart from simply remembering which keys to press!
Do you have any tips on how to improve and become more fluent? Or is it just a matter of forcing yourself to use it until you're fast enough?

**Comment by pascalkuthe** [comment]
Getting used to modal editors simply takes a while that's normal. A few hours isn't much at all if you never used a modal editor. Helix tends to be a lot easier to learn then other modal editor (helpful key prompts, selection -> action, mnemonic keymap) so just stick with it for a while. Be sure to complete the tutor first tough that helps

**Comment by aclueless** [comment]
I am reading the [Insert Mode section here](https://docs.helix-editor.com/keymap.html#insert-mode). It says that:

    💡 New users are strongly encouraged to learn the modal editing paradigm to get the smoothest experience.

But I am completely new to helix and modal editors. I have no idea where should I go to *learn the modal editing paradigm*. It is very helpful if the mentioned text has a link pointing to some pages that I can learn about it.
